page-objects/lib/toast.po.ts

```diff
@@ -0,0 +1,33 @@
+import type {Locator} from '@playwright/test';
+import {BasePO} from '@agnos-ui/base-po';
+
+export const toastSelectors = {
+	rootComponent: '.toast',
+	closeButton: '.btn-close',
+	header: '.toast-header',
+	body: '.toast-body',
+};
+
+export class ToastPO extends BasePO {
+	selectors = structuredClone(toastSelectors);
+
+	override getComponentSelector(): string {
+		return this.selectors.rootComponent;
+	}
+
+	get locatorCloseButton(): Locator {
+		return this.locatorRoot.locator(this.selectors.closeButton);
+	}
+
+	async state() {
+		return await this.locatorRoot.evaluate((rootNode: HTMLElement, selectors) => {
+			const body = rootNode.querySelector<HTMLDivElement>(selectors.body)?.innerText;
+			const header = rootNode.querySelector<HTMLDivElement>(selectors.header)?.innerText;
+			return {
+				rootClasses: rootNode.className.trim().split(' ').sort(),
+				body,
+				header,
```

Comment:
You can also add if th close button is here, its aria-label attribute I think

Reply:
makes sense to add it in the state, as we have the button as part of the component